qa/L0_inferentia_perf_analyzer/test.sh

```diff
@@ -31,7 +31,7 @@ if [ ${USE_TENSORFLOW} == "1" ] && [ ${USE_PYTORCH} == "1" ] ; then
     exit 0
 elif [ ${USE_TENSORFLOW} == "1" ] ; then
     echo "Setting up enviroment with tensorflow 1"
-    source ${TRITON_PATH}/python_backend/inferentia/scripts/setup.sh -t --tensorflow-version 1
```

Comment:
The tensorflow version here is tensorflow-neuron from AWS Neuron SDK. You should not make any changes to TF version here in this file.

Reply:
I see. Reverted TF version changes in this file.